test/metabase/api/database_test.clj

```diff
@@ -237,7 +237,7 @@
                                                  :table_id           (id :categories)
                                                  :special_type       "id"
                                                  :name               "ID"
-                                                 :display_name       "Id"
+                                                 :display_name       "ID"
```

Comment:
and just to be clear, i'm fine with it this way, i just want to make sure.  I assume we picked `Id` originally for a reason.